src/sparseml/yolov8/trainers.py

```diff
@@ -267,11 +268,26 @@ def _setup_train(self, rank, world_size):
                 config["manager"] = str(self.manager)
             loggers = [PythonLogger(logger=LOGGER)]
             try:
-                loggers.append(WANDBLogger(init_kwargs=dict(config=config)))
+                init_kwargs = dict(config=config)
+                if self.args.project is not None:
+                    init_kwargs["project"] = self.args.project
+                if self.args.name is not None:
+                    init_kwargs["name"] = self.args.name
+                loggers.append(WANDBLogger(init_kwargs=init_kwargs))
             except ImportError:
                 warnings.warn("Unable to import wandb for logging")
             self.logger_manager = LoggerManager(loggers)
 
+        if self.args.recipe is not None:
+            base_path = os.path.join(self.save_dir, "original_recipe.yaml")
+            with open(base_path, "w") as fp:
+                fp.write(load_recipe_yaml_str(self.args.recipe))
```

Comment:
might be good to move this to a more generic helper, could see this being useful in more integrations

Reply:
Yeah I copied it from torchvision integration, so +1 for later